packages/docs-site/src/library/pages/components/form/input.md

```diff
@@ -35,17 +35,17 @@ The Text Input should stand out and be easily discoverable by users. The text la
 ### Anatomy
 <InputAnatomy />
 
-1. **Label**. The Label should be used to describe to the user what the desired input should be. Every field should have a text label. The label should always be visible on all inputs, excluding search bars.
-2. **User Input**. The User Input is the text the user has entered into the Text Input.
-3. **Container**. The Container wraps the entire component.
+1. **Label**. Should be used to describe to the user what the desired input should be. Every field should have a text label. The label should always be visible on all inputs, excluding search bars.
+2. **User Input**. This is the text the user has entered into the Text Input.
+3. **Container**. Wraps the entire component.
 
 ### Sizing & Spacing
-The Text Input is available in 1 standard size. It has been created in relation to other form elements to ensure consistency.
+The Text Input is available in one standard size. It has been created in relation to other form elements to ensure consistency.
 
 ### States
 <InputStates />
 
-The Text Input has 3 states - default, active, and filled.
+The Text Input has three states - default, active, and filled.
```

Comment:
Could back ticks be used here around the states?

Reply:
Good spot. Have updated all instances of the Component State sections to have back ticks around the states.